Program.cs

```diff
@@ -138,15 +215,43 @@ public static void Main(string[] args)
                     .WithDefaultSubscription();
 
                 // Print selected subscription
-                Utilities.Log("Selected subscription: " + azure.SubscriptionId);
+                Console.WriteLine("Selected subscription: " + azure.SubscriptionId);
 
                 RunSample(azure);
             }
             catch (Exception e)
             {
-                Utilities.Log(e.ToString());
+                Console.WriteLine(e.ToString());
             }
         }
+
+        public static void PrintSubscriptionsInTopic(Microsoft.Azure.Management.ServiceBus.Fluent.ISubscription serviceBusSubscription)

```

Comment:
This need not be public right? Also please add similar methods to print queues, topics and the namespace

Reply:
@ramya-rao-a I have updated, please review.